manager/pom.xml

```diff
@@ -198,6 +199,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.aliyun.oss</groupId>
+            <artifactId>aliyun-sdk-oss</artifactId>
+            <version>${alibaba.oss.version}</version>
+        </dependency>
```

Comment:
hi, can we use http api instead of `aliyun sdk`, the sdk will import many irrelevant dependencies may also cause compatibility issues between dependency packages and licenses.  See https://github.com/apache/hertzbeat/tree/master/material/licenses.  We plan to remove the dependency on the cloud service SDK like `aliyun, tencent, huawei sdk` later.

Reply:
ok，we can change it